doc/sphinx-guides/source/installation/oauth2.rst

```diff
@@ -11,7 +11,7 @@ As explained under "Auth Modes" in the :doc:`config` section, OAuth2 is one of t
 
 `OAuth2 <https://oauth.net/2/>`_ is an authentication protocol that allows systems to share user data, while letting the users control what data is being shared. When you see buttons stating "login with Google" or "login through Facebook", OAuth2 is probably involved. For the purposes of this section, we will shorten "OAuth2" to just "OAuth." OAuth can be compared and contrasted with :doc:`shibboleth`.
 
-Dataverse supports three OAuth providers: `ORCID <http://orcid.org>`_, `GitHub <https://github.com>`_, and `Google <https://console.developers.google.com>`_.
```

Comment:
@djbrooke should we add "Microsoft" to the title of this page? Right now line 1 is "OAuth Login: ORCID, GitHub, Google".

Reply:
Updated in https://github.com/IQSS/dataverse/pull/6192/commits/1bf38c2ee36dec99e667289198ea768228519924